src/js/lib/renderingFilter.js

```diff
@@ -0,0 +1,44 @@
+const filter = (items, filterType) => {
+  const filtering = {
+    active: () => {
+      return items.filter((item) => {
+        return !item.isCompleted;
+      });
+    },
+    completed: () => {
+      return items.filter((item) => {
+        return item.isCompleted;
+      });
+    },
+    all: () => {
+      return items;
+    },
+    priority: () => {
+      // let cloneItems = JSON.parse(JSON.stringify(items));
+
+      let priority0 = [];
+      let priority12 = [];
+
+      items.forEach((item) => {
+        if (item.priority === 0) {
+          return priority0.push(item);
+        }
+        priority12.push(item);
+      });
+
+      priority12.sort((a, b) => {
+        return a.priority - b.priority;
+      });
+
+      console.log(priority0);
+      console.log(priority12);
+      const priorityItems = priority12.concat(priority0);
```

Comment:
제가 전체적인 구조를 파악하진 못해 섣불리 말씀드리긴 힘들지만, 우선순위를 지정하지 않았을 때의 값을 3 이상의 수로 설정하면 concat을 쓰지 않고도 코드 수를 다음과 같이 줄일 수 있을 것 같아요.
```suggestion
      const priorityItems = items.slice();
      priorityItems.sort((a, b) => a.priority - b.priority);
```

Reply:
저도 처음에 이렇게 구현해볼까 생각을 했었는데 확장성 때문에.. 혹시나 우선순위가 점점 늘어나게 되면 나중에 임의로 숫자를 계속 지정해줘야 할 것 같아서 다른 방법으로 해보려고 했습니다. 하지만 이번 프로젝트는 우선순위가 2까지밖에 없으니 간결한 코드를 위해 임의로 3이상의 수를 지정해줬어도 좋았을 것 같네요 !